en/option/component/axis-common.md

```diff
@@ -147,6 +147,54 @@ Whether to show the label of the min tick. Optional values: `true`, `false`, `nu
 
 Whether to show the label of the max tick. Optional values: `true`, `false`, `null`. It is auto determined by default, that is, if labels are overlapped, the label of the max tick will not be displayed.
 
+{{ if: ${componentType} === 'xAxis' }}
+
+##${prefix} alignMinLabel(string) = null
```

Comment:
Just a reminder. Don't forget to add a version support tip.

Reply:
Thanks~